decentraland/unity-explorer
# add scene restriction minimap badge

## What does this PR change?



Fix #1809 

Introduce the scene restrictions icon in the minimap, indicating which avatar restrictions are applied while you are in the scene.

The original issue mentions 2 restrictions that aren't currently handled by E@:

1. Passports can not be opened
2. Experiences are blocked

These are to be handled in the restrictions icon once they are implemented.

## How to test the changes?



1. Launch the explorer
4. Go to mini games that restrict your avatar (movements, camera, ...)
5. Check that once you are playing, right beside your coordinates on the minimap, a warning icon appears and if you hover it with your mouse, a dialog with all the restrictions appears
6. Check that the restriction listed in the toast message are consistent with the actual applied restrictions
7. Check that if you leave the mini game (either by teleporting or exiting it), the icon is reset

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
